events-api/src/modules/tickets/tickets.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface TicketsModel {
+  ticketId: string;
+  ticket_quantity: number;
```

Comment:
Use consistent camelCase

Reply:
![image](https://github.com/user-attachments/assets/ad1d7481-d881-4789-add1-8f4535976539)

Reply:
Please be consistent with API response casing too. Check the API responses

Reply:
Also just noticed, please use singular name for model name. Please fix it

Reply:
fixed